src/openrct2/peep/Peep.cpp

```diff
@@ -474,13 +474,14 @@ std::optional<CoordsXY> Peep::UpdateAction(int16_t& xy_distance)
     WindowInvalidateFlags |= PEEP_INVALIDATE_PEEP_2;
 
     // Create sick at location
```

Comment:
```suggestion
```

It’s clear from the code anyway, and it’s a colloquial usage of the word that won’t be familiar to everyone.